CLI Release updates (#3960)
astronomer/docs: astro/cli/astro-deployment-update.md

| ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `<deployment-id>` (_Required_) | The ID of the Deployment to update                                                                                                                                                   | Any valid Deployment ID                                                                                                                                                                                                             |

> | `-c`, `--cicd-enforcement`           | Specify that the Deployment can only accept code deploys from API tokens and keys. Note that in CLI versions before 1.23, this flag was `--enforce-cicd`.                            | None                                                                                                                                                                                                                                |

Reviewer: This is incorrect. There is no shorthand flag for cicd-enforcement. 

```suggestion
| `--cicd-enforcement`           | Specify that the Deployment can only accept code deploys from API tokens and keys. Note that in CLI versions before 1.23, this flag was `--enforce-cicd`.                            | None                                                                                                                                                                                                                                |
```

|
| `--deployment-file`            | Location of the Deployment file to update the Deployment with. The file format can be JSON or YAML. See [Create a Deployment with a Deployment File](manage-deployments-as-code.md). | A valid file path to any YAML or JSON Deployment file                                                                                                                                                                               |
| `--deployment-name`            | The name of the Deployment to update. Use as an alternative to `<deployment-id>`.                                                                                                    | Any valid Deployment name                                                                                                                                                                                                           |

> | `-d`,`--description`           | The description for the Deployment                                                                                                                                                   | Any string. Multiple-word descriptions should be specified in quotations (`"`)                                                                                                                                                      |

Reviewer 2: ```suggestion
| `-d`,`--description`           | A description of the Deployment                                                                                                                                                   | Any string. Multiple-word descriptions should be specified in quotations (`"`)                                                                                                                                                      |
```

> |`-m`, `--development-mode` |   Whether the Deployment is for development mode only. If set to `disable`, the Deployment can be considered production for the purposes of Astro support case priority, but development-only features such as hibernation aren't available. You can't update this value to `enable` for existing non-development Deployments. Set to `disable` by default | `disable` or `enable` |

Reviewer 2: ```suggestion
|`-m`, `--development-mode` |   Whether the Deployment is for development mode only. If set to `disable`, the Deployment can be considered production for the purposes of Astro support case priority, but development-only features such as hibernation aren't available. You can't update this value to `enable` for existing non-development Deployments. Set to `disable` by default. | `disable` or `enable` |
```

| `-e`,`--executor`              | The executor to use for the Deployment                                                                                                                                               | CeleryExecutor or KubernetesExecutor                                                                                                                                                                                                |
| `-f`,`--force`                 | Force a Deployment update                                                                                                                                                            | None                                                                                                                                                                                                                                |

> | `-n`,`--name`                  | The Deployment's name                                                                                                                                                                | Any string. Multiple-word descriptions should be specified in quotations                                                                                                                                                            |

Reviewer 2: ```suggestion
| `-n`,`--name`                  | The Deployment's name                                                                                                                                                                | Any string. Multiple-word descriptions should be specified in quotations.                                                                                                                                                            |
```

| `--resource-quota-cpu`         | (Astro Hosted only) The Deployment's CPU resource quota for Kubernetes Pods. See [Customize a task's Kubernetes Pod](kubernetes-executor.md#customize-a-tasks-kubernetes-pod) for more information. |    A numeric value such as `10`. This number cannot exceed the values you configured in your [Default pod resources](deployment-resources.md#configure-kubernetes-pod-resources) in the UI or in your [Deployment file](deployment-file-reference.md) configuration file.                                                    |
| `--resource-quota-memory`         | (Astro Hosted only) The Deployment's memory resource quota for Kubernetes Pods in Gi. See [Customize a task's Kubernetes Pod](kubernetes-executor.md#customize-a-tasks-kubernetes-pod) for more information. |    A numeric value followed by `Gi`, such as `20Gi`. This number cannot exceed the values you configured in your [Default pod resources](deployment-resources.md#configure-kubernetes-pod-resources) in the UI or in your [Deployment file](deployment-file-reference.md) configuration file.                                                    |

> | `--scheduler-size` | The size of the Scheduler for the Deployment. | Possible values can be `small`, `medium`, or `large`|

Reviewer 2: ```suggestion
| `--scheduler-size` | The size of the Scheduler for the Deployment. | Possible values can be `small`, `medium`, or `large`. |
```
